Source/WebGPU/WebGPU/PresentationContextIOSurface.h

```diff
@@ -65,6 +66,9 @@ class PresentationContextIOSurface : public PresentationContext {
     size_t m_currentIndex { 0 };
     id<MTLFunction> m_luminanceClampFunction;
     id<MTLComputePipelineState> m_computePipelineState;
+#if HAVE(IOSURFACE_SET_OWNERSHIP_IDENTITY) && HAVE(TASK_IDENTITY_TOKEN)
+    const Instance& m_instance;
```

Comment:
Raw reference here too, should at least be CheckedRef or something. How do we ensure all of these are destroyed before the creating `Instance`?